### DISCUSSION TITLE
bull 定时任务执行完成后，重复又执行了一次

### DISCUSSION THREAD

**Discussion by mynamexjl** [midwayjs/midway#3730] [Q&A]
### Describe the problem（描述问题）

报错日志：
![image](https://github.com/midwayjs/midway/assets/25628679/b008327e-322b-4b34-8d93-08997da4e486)
因为上次error打印问题，退到了3.11.6版本。  redis 还是那种集群式 魔改版

### Midway Versions（Midway 版本）

 "@midwayjs/bull": "3.11.6"

**Comment by czy88840616** [comment]
事务问题吧，出错了自动重试了？

**Reply by mynamexjl** [reply]
没有用到事务，这个现象也是偶发的 并不是每次都能触发。 服务可能运行10来天都没有问题，就是有天就是会出现这个报错。出现了这个报错后，正在跑的队列又会重新入队，再次执行
